src/components/my-career/CategoryCard.jsx

```diff
@@ -1,12 +1,10 @@
-import {
-  useEffect, useMemo, useState,
-} from 'react';
+import { useEffect, useMemo, useState } from 'react';
 import PropTypes from 'prop-types';
 import { Button, Card, useToggle } from '@openedx/paragon';
-import { getConfig } from '@edx/frontend-platform/config';
-import algoliasearch from 'algoliasearch/lite';
 import { FormattedMessage } from '@edx/frontend-platform/i18n';
 
+import algoliasearch from 'algoliasearch/lite';
+import { getConfig } from '@edx/frontend-platform/config';
```

Comment:
nit: these imports are external deps so should be grouped with the the above imports, not the local imports below.